src/main/resources/com/puppycrawl/tools/checkstyle/grammars/javadoc/JavadocParser.g4

```diff
@@ -908,7 +910,9 @@ javadocInlineTag:
             | LINKPLAIN_LITERAL (WS | NEWLINE | LEADING_ASTERISK)* reference description?
             | LITERAL_LITERAL (WS | NEWLINE | LEADING_ASTERISK | text)*
             | VALUE_LITERAL (WS | NEWLINE | LEADING_ASTERISK)* reference?
-            | CUSTOM_NAME (WS | NEWLINE | LEADING_ASTERISK)* description?
+            | CUSTOM_NAME (WS | NEWLINE | LEADING_ASTERISK)*
+              ({!isNextJavadocTag()}? LEADING_ASTERISK | htmlComment
+              | CDATA | NEWLINE | text | javadocInlineTag | htmlElement)*
```

Comment:
So, here's the thing. The javadoc tool from oracle doesn't seem to support inline custom javadoc tags _(correct me if I am wrong)_ without writing a custom taglet/doclet. But what if some other tool does or a user is writing his/her own taglet/doclet. Be it a block tag or an inline tag, I suggest that we our ASTs should represent everything that's standard rather than representing everything as text since that's what a user would expect us to ? What do you guys think ?